cmd/plugin_test.go

```diff
@@ -362,6 +362,175 @@ func TestPluginMulti(t *testing.T) {
 	}
 }
 
+// Test recursive downloads from the plugin
+func TestPluginRecursiveDownload(t *testing.T) {
+	viper.Reset()
+	server_utils.ResetOriginExports()
+
+	dirName := t.TempDir()
+
+	viper.Set("Logging.Level", "debug")
+	viper.Set("Origin.StorageType", "posix")
+	viper.Set("Origin.ExportVolumes", "/test")
```

Comment:
This really isn't how the `ExportVolumes` is meant to be used, and it is malformed because export volumes are meant to contain both a storage and a federation prefix in posix. If you're trying to set up a federation prefix, you should use `Origin.FederationPrefix`, and if this is meant to be a storage prefix, you should use `Origin.StoragePrefix`. The export volumes variable is really only meant to be configured via env vars or bound to the command line `-v` when serving origins.